examples/handlebars_templates/src/main.rs

```diff
@@ -42,7 +43,21 @@ fn not_found(req: &Request) -> Template {
 fn rocket() -> rocket::Rocket {
     rocket::ignite()
         .mount("/", routes![index, get])
-        .attach(Template::fairing())
+        .attach(Template::custom(|engines| {
```

Comment:
This looks pretty ugly. Can we factor the helper out into a function and then just call `register_helper("echo", Box::new(echo_help))` here?